### PR TITLE
Add github issue template

### PR DESCRIPTION
**Pull request by jedireza**
Adding a simple GitHub issue template.

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+NOTE: If this is a security or privacy related issue, please email
+incidents@neeva.co instead of submitting an issue on GitHub.
+
+Thank you for submitting an issue. Provide as much information as you can.
+
+## Expected behavior
+
+This went wrong.
```
Comment (darinwf) [inline .github/issue_template.md:8]: nit: sections are flipped

**Review by darinwf — APPROVED** [review]
Thanks!

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+NOTE: If this is a security or privacy related issue, please email
+incidents@neeva.co instead of submitting an issue on GitHub.
+
+Thank you for submitting an issue. Provide as much information as you can.
+
+## Expected behavior
```
Comment (EdwardJTL) [inline .github/issue_template.md:6]: nit: the all titles should follow the same capitalization rule

Comment (jedireza) [reply .github/issue_template.md:6]: Thanks! I'm so picky about this in general. I'm surprised I missed it. 😅

**Pull request merged**